tests/21presence-events.pl

```diff
@@ -1,108 +0,0 @@
-# Eventually this will be changed; see SPEC-53
-my $PRESENCE_LIST_URI = "/r0/presence/list/:user_id";
-
-
-test "initialSync sees my presence status",
```

Comment:
this doesn't seem to be related to presence lists?

Reply:
yup saw the same - sec